cPP_DSC.adoc

```diff
@@ -683,9 +685,7 @@ FCS_COP.1/Hash Cryptographic Operation - Hashing
 
 FCS_COP.1.1/Hash:: The TSF shall perform [_cryptographic hashing_] in accordance with a specified cryptographic algorithm [*selection*: _SHA-1, SHA-224, SHA-256, SHA-384, SHA-512, SHA-512/224, SHA-512/256, SHA3-224, SHA3-256, SHA3-384, SHA3-512_] that meets the following: [*selection*: _ISO/IEC 10118-3:2018 [SHA, SHA3], FIPS PUB 180-4 [SHA], FIPS PUB 202 [SHA3]_].
 
-_Application Note {counter:remark_count}_:: _The hash selection shall be consistent with the overall strength of the algorithm used for signature generation. For example, the TOE should choose SHA-256 for 2048-bit RSA or ECC with P-256; SHA-384 for 3072-bit RSA, 4096-bit RSA, or ECC with P-384; and SHA-512 for ECC with P-521. The ST author selects the standard based on the algorithms selected._
```

Comment:
Is there any replacement for this sentence?

Reply:
No, I just took it out. There is added text at the beginning of the crypto section that says that the key strength should be commensurate with the overall level, and that's basically what this is saying. The shall statement is basically trying to enforce something that really can't be enforced unless we actually make a specific SFR that would require the vendor to define it up front, so this is really general guidance, but the "shall" is a problem. I couldn't see how to really do it, and then keep doing it over and over where this is stated in different ways for different requirements, I counted 3 so far.